features/api/mandate.feature

```diff
@@ -9,37 +9,145 @@ Feature: Mandates management
   New jobs are created for the current mandate.
   A user may have one or several mandate, with or without a job.
 
-#  Scenario: If one list all the mandates, there it at least one mandate: the current one.
-#    #TODO
-#
-#  Scenario: A mandate may have users.
-#    #TODO
-#
-#  Scenario: It should be possible to list all the mandates.
-#    #TODO
-#
-#  Scenario: It should be possible to list all the members of a given mandate.
-#    #TODO
-#
-#  Scenario: It should not be possible to create a mandate unless it starts at the current year. The ending date
-#  should then be during the next year and may be omitted.
-#    #TODO
-#
-#  Scenario: If not ending date is given for a mandate, it ends at the end of the next year.
-#    #TODO
-#
-#  Scenario: If no new mandate has be created, a new one is automatically created and keep all the admin users as if
-#  they have a new mandate.
-#    #TODO
-#
-#  Scenario: It should not be possible to delete a mandate.
-#    #TODO
-#
-#  Scenario: A new member can be added to a mandate even if the mandate already ended.
-#    #TODO
-#
-#  Scenario: A member may be deleted from a mandate even if the mandate already ended.
-#    #TODO
-#
+  Scenario: If one list all the mandates, there it at least one mandate: the current one.
+    #TODO
+
+
+  Scenario: It should be possible to list all the mandates.
+    #TODO
+
+
+#START -----Filter validation-----
+  Scenario: It should be possible to list all the members of a given mandate.
+    #TODO
+
+#END -----Filter validation-----
+
+#START -----Crud validation-----
+
+  Scenario: If no new mandate has be created, a new one is automatically created and keep all the admin users as if they have a new mandate.
+    #TODO
+
+  Scenario: It should be possible to create a new mandate
+   #TODO
+
+  Scenario: It should be possible to update mandate
+   #TODO
+
+  Scenario: It should not be possible to delete a mandate.
+    #TODO
+#END -----Crud validation-----
+
+#START -----Spec validation-----
+
+  Scenario: A mandate may have users.
+    #TODO
+
+  Scenario: It should not be possible to create a mandate unless it starts at the current year. The ending date
+  should then be during the next year and may be omitted.
+    #TODO
+
+  Scenario: If not ending date is given for a mandate, it ends at the end of the next year.
+    #TODO
+
+  Scenario: A new member can be added to a mandate even if the mandate already ended.
+    #TODO
+
+  Scenario: A member may be deleted from a mandate even if the mandate already ended.
+   #TODO
+
 #  Scenario: Once a mandate created, the dates may change but must be of the same year.
-#    #TODO
+	#TODO
+#END -----Spec validation-----
```

Comment:
Why is the following partly implemented? I mean this should be kept rather than the TODO